### PR TITLE
#157906013 Custom theme

### PR DESCRIPTION
**Pull request by chuzksy-codeactive**
#### What does this PR do?
Create a new reaction theme for shop

#### Description of Task to be completed?
Users should see a new theme when logged into the app and when they receive an email from the vendor
#### How should this be manually tested?
clone repo, install package dependencies by running: `meteor npm install` on the command prompt
Then run `reaction` or `reaction run`

#### Screenshots (if appropriate)
screen shot 2018-06-04 at 5 25 00 am
The mage above shows how each email template would look like. Note that the sample colours provided in the PT user story are applied here.

### CODE REVIEW

File: imports/plugins/custom/thorinrc-theme/client/index.less [hunk 1]
```diff
@@ -0,0 +1 @@
+@import "styles/navbar.less";
```
Comment (emmabaye) [inline imports/plugins/custom/thorinrc-theme/client/index.less]: There should be a new line at the end of this file.

**Review by emmabaye — COMMENTED** [review]
Nice UI. It would have been better if this PR was deployed.

File: imports/plugins/custom/thorinrc-theme/client/index.less (hunk 1 above)
Comment (chuzksy-codeactive) [reply imports/plugins/custom/thorinrc-theme/client/index.less]: @emmabaye Thank you for your review, for the deployment @amarachukwu-agbo  is working on it. And I have added a new line.

**Review by Orlayhemmy — COMMENTED** [review]
Good Job

**Pull request merged**